workos/resources/base.py

```diff
@@ -0,0 +1,36 @@
+class WorkOSBaseResource(object):
+    """Representation of a WorkOS Resource as returned through the API.
+    
+    Attributes:
+        OBJECT_FIELDS (list): List of fields a Resource is comprised of.
+    """
+
+    OBJECT_FIELDS = []
```

Comment:
hmm, should this be a class variable, as opposed to an instance variable instead?